packages/flutter/lib/src/cupertino/tab_view.dart

```diff
@@ -57,6 +57,15 @@ class CupertinoTabView extends StatefulWidget {
   ///
   /// If a [builder] is specified, then [routes] must not include an entry for `/`,
   /// as [builder] takes its place.
+  ///
+  /// Updating the [builder]'s existing [WidgetBuilder] instance will not clear
```

Comment:
nits:
Rebuilding a [CupertinoTabView] with a different [builder] will not clear its current navigation stack or update its descendants. Instead, trigger a rebuild from a widget lower in its subtree. This can be done via methods such as:

  * Calling [State.setState] on a descendant [StatefulWidget]'s [State] object.
  * Modifying an [InheritedWidget] from which a descendant has inherited.

Reply:
Done with some tweaks. We're not technically triggering a rebuild from a 'widget' since Widget instances can't 'do' anything. 

'from which a descendant has inherited' is technically right but might be a bit vague.